src/qml/AboutPage.qml

```diff
@@ -29,6 +29,28 @@ Flickable {
         id: diskUsage
     }
 
+    Item {
+        id: uptimeCounter
+        property int totaltime : uptime
+
+        Timer {
+            interval: 1000
+            repeat: true
+            triggeredOnStart: true
+            running: true
+            onTriggered: ++uptimeCounter.totaltime
+        }
+        function days() {
+            return Math.floor(uptimeCounter.totaltime / 86400)
+        }
+        function asString() {
+            var now = uptimeCounter.totaltime
+            var days = Math.floor(now / 86400)
```

Comment:
I deliberately avoided calling days because it's possible for the timer to update between the call to initialize `now` and rest of the calculation.  If `now` was called at an uptime of 23:59:59,  `days()` might get the second after that and then the display would incorrectly show "0 days 00:00:00".

Reply:
And to clarify, having the uptime off by 30 seconds doesn't seem likely to matter much, but having it off by a day probably does.